README.md

```diff
@@ -19,6 +19,7 @@ A non-exhaustive list of projects to be known to have rather unpleasant vibes.
 * [Slimefun](https://github.com/Slimefun/Slimefun4) "It offers everything you could possibly imagine. From taking 70% of your tick to clearing inventories at random!" - Tries to turn your server in an FTB modpack, and does not achieve its goal due to developers not knowing what they are doing.
 * [FastAsyncWorldEdit](https://www.spigotmc.org/resources/fast-async-worldedit.13932) "It's async, so that means it's better!" - Aims to be a fast fork of WorldEdit, it sure is fast, fast at breaking things.
 * [IllegalStack](https://www.spigotmc.org/resources/dupe-fixes-illegal-stack-remover.44411/) "Dupe & Exploit fixer" - Actually introduces dupes, eats 15% of your ticks, and likes to do wonky things like removing nametags of players and entities.
+* [Skript](https://github.com/SkriptLang/Skript) - "Programming language, that allows server owners to modify their servers without learning Java" - Widely used preformance killer by people, who belive that putting 2 phrases next to each other in Skript makes them a developer. Not to mention the thousands of [forks](https://skunity.com/), claimimg to "improve preformance".
```

Comment:
```suggestion
* [Skript](https://github.com/SkriptLang/Skript) - "Programming language that allows server owners to modify their servers without learning Java" - Widely used preformance killer by people, who believe that putting 2 phrases next to each other in Skript makes them a developer. Not to mention the thousands of [forks](https://skunity.com/), claimimg to "improve preformance".
```

Reply:
I personally would go with `Wanna-be Programming language` as I can't even see skript as actual programming and more as scripting itself.
Pretty sure other Java devs would agree on this.